[Bug] [FLUX] Meu Treino está dando erro: "<uzywajqzbdbrfammshdg...

## Bug Report

**Reported by:** Anonymous
**Category:** Bug
**URL:** `https://aica.guru/meu-treino`
**Timestamp:** 2026-03-05T18:13:03.262Z

---

### Description

[FLUX] Meu Treino está dando erro: "<uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_INTERNET_DISCONNECTED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at vendor-supabase-CbDNTCDZ.js:27:3851
    at vendor-supabase-CbDNTCDZ.js:27:3922
    at new Promise (<anonymous>)
    at us (vendor-supabase-CbDNTCDZ.js:27:3780)
    at fn._refreshAccessToken (vendor-supabase-CbDNTCDZ.js:43:19765)
    at fn._callRefreshToken (vendor-supabase-CbDNTCDZ.js:43:22637)
console.error @ widget.js:14
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_INTERNET_DISCONNECTED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
withcallout.com/api/onboarding?project=lc-qjc-zO8Z8&url=https%3A%2F%2Faica.guru%2F&anonymous=true:1  Failed to load resource: net::ERR_INTERNET_DISCONNECTED
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_NAME_NOT_RESOLVED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_NETWORK_CHANGED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
widget.js:14 AuthRetryableFetchError: Failed to fetch
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9241)
    at async k (vendor-supabase-CbDNTCDZ.js:27:8977)
    at async vendor-supabase-CbDNTCDZ.js:43:19851
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
main-DQ0H7cAg.js:2 @supabase/gotrue-js: Lock "lock:sb-uzywajqzbdbrfammshdg-auth-token" was not released within 5000ms. This may indicate an orphaned lock from a component unmount (e.g., React Strict Mode). Forcefully acquiring the lock to recover.
Hs.console.warn @ main-DQ0H7cAg.js:2
main-DQ0H7cAg.js:2 @supabase/gotrue-js: Lock "lock:sb-uzywajqzbdbrfammshdg-auth-token" was not released within 5000ms. This may indicate an orphaned lock from a component unmount (e.g., React Strict Mode). Forcefully acquiring the lock to recover.
Hs.console.warn @ main-DQ0H7cAg.js:2
main-DQ0H7cAg.js:2 @supabase/gotrue-js: Lock "lock:sb-uzywajqzbdbrfammshdg-auth-token" was not released within 5000ms. This may indicate an orphaned lock from a component unmount (e.g., React Strict Mode). Forcefully acquiring the lock to recover.
Hs.console.warn @ main-DQ0H7cAg.js:2
widget.js:14 [AICA] Unhandled promise rejection: AbortError: Lock broken by another request with the 'steal' option.
console.error @ widget.js:14
vendor-supabase-CbDNTCDZ.js:43 Uncaught (in promise) AbortError: Lock broken by another request with the 'steal' option.
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_NETWORK_CHANGED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at vendor-supabase-CbDNTCDZ.js:27:3851
    at vendor-supabase-CbDNTCDZ.js:27:3922
    at new Promise (<anonymous>)
    at us (vendor-supabase-CbDNTCDZ.js:27:3780)
    at fn._refreshAccessToken (vendor-supabase-CbDNTCDZ.js:43:19765)
    at fn._callRefreshToken (vendor-supabase-CbDNTCDZ.js:43:22637)
console.error @ widget.js:14
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_QUIC_PROTOCOL_ERROR
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
uzywajqzbdbrfammshdg.supabase.co/auth/v1/token?grant_type=refresh_token:1  Failed to load resource: net::ERR_NAME_NOT_RESOLVED
widget.js:14 TypeError: Failed to fetch
    at vendor-supabase-CbDNTCDZ.js:27:2946
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9184)
    at k (vendor-supabase-CbDNTCDZ.js:27:8983)
    at vendor-supabase-CbDNTCDZ.js:43:19857
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
widget.js:14 AuthRetryableFetchError: Failed to fetch
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9241)
    at async k (vendor-supabase-CbDNTCDZ.js:27:8977)
    at async vendor-supabase-CbDNTCDZ.js:43:19851
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
widget.js:14 AuthRetryableFetchError: Failed to fetch
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9241)
    at async k (vendor-supabase-CbDNTCDZ.js:27:8977)
    at async vendor-supabase-CbDNTCDZ.js:43:19851
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
widget.js:14 AuthRetryableFetchError: Failed to fetch
    at Ss (vendor-supabase-CbDNTCDZ.js:27:9241)
    at async k (vendor-supabase-CbDNTCDZ.js:27:8977)
    at async vendor-supabase-CbDNTCDZ.js:43:19851
    at async vendor-supabase-CbDNTCDZ.js:27:3845
console.error @ widget.js:14
widget.js:14 ❌ [useAuth] Error getting session: Object
console.error @ widget.js:14
main-DQ0H7cAg.js:2 ⚠️ [GoogleCalendarTokenService] [getValidAccessToken] Token EXPIRED. Attempting refresh...
Hs.console.warn @ main-DQ0H7cAg.js:2
main-DQ0H7cAg.js:2 ⚠️ [GoogleCalendarTokenService] [getValidAccessToken] Token EXPIRED. Attempting refresh...
Hs.console.warn @ main-DQ0H7cAg.js:2
widget.js:14 Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981)
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
pa @ vendor-react-Tn1wb__B.js:40
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 ❌ [ErrorBoundary] ErrorBoundary caught an error: Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981) {componentStack: '\n    at ta (https://aica.guru/assets/AthletePortal…ca.guru/assets/vendor-react-Tn1wb__B.js:51:24235)'}
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
ce @ services-BKjl_T6O.js:1
error @ services-BKjl_T6O.js:1
componentDidCatch @ module-journey-BQI7qcGj.js:13
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981)
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
pa @ vendor-react-Tn1wb__B.js:40
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 ❌ [ErrorBoundary] ErrorBoundary caught an error: Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981) {componentStack: '\n    at ta (https://aica.guru/assets/AthletePortal…ca.guru/assets/vendor-react-Tn1wb__B.js:51:24235)'}
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
ce @ services-BKjl_T6O.js:1
error @ services-BKjl_T6O.js:1
componentDidCatch @ module-journey-BQI7qcGj.js:13
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981)
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
pa @ vendor-react-Tn1wb__B.js:40
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 ❌ [ErrorBoundary] ErrorBoundary caught an error: Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981) {componentStack: '\n    at ta (https://aica.guru/assets/AthletePortal…ca.guru/assets/vendor-react-Tn1wb__B.js:51:24235)'}
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
ce @ services-BKjl_T6O.js:1
error @ services-BKjl_T6O.js:1
componentDidCatch @ module-journey-BQI7qcGj.js:13
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981)
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
pa @ vendor-react-Tn1wb__B.js:40
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
widget.js:14 ❌ [ErrorBoundary] ErrorBoundary caught an error: Error: Minified React error #310; visit https://reactjs.org/docs/error-decoder.html?invariant=310 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at xt (vendor-react-Tn1wb__B.js:38:17785)
    at Object.pc [as useMemo] (vendor-react-Tn1wb__B.js:38:21480)
    at fe.useMemo (vendor-react-Tn1wb__B.js:9:6377)
    at ta (AthletePortalView-seTo6EhC.js:9:29945)
    at oa (vendor-react-Tn1wb__B.js:38:17190)
    at ha (vendor-react-Tn1wb__B.js:40:3158)
    at i1 (vendor-react-Tn1wb__B.js:40:45080)
    at n1 (vendor-react-Tn1wb__B.js:40:39973)
    at I0 (vendor-react-Tn1wb__B.js:40:39901)
    at Sl (vendor-react-Tn1wb__B.js:40:39754)
    at Oa (vendor-react-Tn1wb__B.js:40:36083)
    at Xc (vendor-react-Tn1wb__B.js:40:35031)
    at te (vendor-react-Tn1wb__B.js:25:1604)
    at MessagePort.He (vendor-react-Tn1wb__B.js:25:1981) {componentStack: '\n    at ta (https://aica.guru/assets/AthletePortal…ca.guru/assets/vendor-react-Tn1wb__B.js:51:24235)'}
console.error @ widget.js:14
Hs.console.error @ main-DQ0H7cAg.js:2
ce @ services-BKjl_T6O.js:1
error @ services-BKjl_T6O.js:1
componentDidCatch @ module-journey-BQI7qcGj.js:13
Sc.c.componentDidCatch.n.callback @ vendor-react-Tn1wb__B.js:40
Ku @ vendor-react-Tn1wb__B.js:38
Zc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
Gc @ vendor-react-Tn1wb__B.js:40
T0 @ vendor-react-Tn1wb__B.js:40
A0 @ vendor-react-Tn1wb__B.js:40
An @ vendor-react-Tn1wb__B.js:40
Xc @ vendor-react-Tn1wb__B.js:40
te @ vendor-react-Tn1wb__B.js:25
He @ vendor-react-Tn1wb__B.js:25
>"

---

### Screenshot

![Screenshot](https://r2pub.withcallout.com/uploads/lc-qjc-zO8Z8/2026-03-05T18-13-04-452Z-fr_rM5mCqYXCYuB0C_fPECyK.webp)

---

### Session Replay

_Session replay not available_

---

### Environment

| Field | Value |
|-------|-------|
| Browser | Chrome 145.0.0.0 |
| OS | Windows |
| Screen Resolution | 1680x1050 |
| Viewport | 1316x997 |

---

### Console Errors

```
[2026-03-05T15:21:42.466Z] {"__isAuthError":true,"name":"AuthRetryableFetchError","status":0}

[2026-03-05T15:21:45.640Z] %c❌ [useAuth] Error getting session: color: #EF4444; font-weight: bold {"error":"Failed to fetch"}

[2026-03-05T18:12:38.045Z] {}

[2026-03-05T18:12:38.047Z] %c❌ [ErrorBoundary] ErrorBoundary caught an error: color: #EF4444; font-weight: bold {} {"componentStack":"\n    at ta (https://aica.guru/assets/AthletePortalView-seTo6EhC.js:9:24786)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at za (https://aica.guru/assets/main-DQ0H7cAg.js:2:27210)\n    at Va (https://aica.guru/assets/main-DQ0H7cAg.js:2:19100)\n    at f (https://aica.guru/assets/main-DQ0H7cAg.js:2:57450)\n    at Gp (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:4073)\n    at N3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:8728)\n    at Suspense\n    at Ma (https://aica.guru/assets/main-DQ0H7cAg.js:2:18658)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at Bs (https://aica.guru/assets/main-DQ0H7cAg.js:2:59470)\n    at va (https://aica.guru/assets/main-DQ0H7cAg.js:2:1882)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Ea (https://aica.guru/assets/main-DQ0H7cAg.js:2:3834)\n    at Us\n    at oh (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:7803)\n    at P3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:17443)\n    at T3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:24235)"}

[2026-03-05T18:12:40.209Z] {}

[2026-03-05T18:12:40.210Z] %c❌ [ErrorBoundary] ErrorBoundary caught an error: color: #EF4444; font-weight: bold {} {"componentStack":"\n    at ta (https://aica.guru/assets/AthletePortalView-seTo6EhC.js:9:24786)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at za (https://aica.guru/assets/main-DQ0H7cAg.js:2:27210)\n    at Va (https://aica.guru/assets/main-DQ0H7cAg.js:2:19100)\n    at f (https://aica.guru/assets/main-DQ0H7cAg.js:2:57450)\n    at Gp (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:4073)\n    at N3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:8728)\n    at Suspense\n    at Ma (https://aica.guru/assets/main-DQ0H7cAg.js:2:18658)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at Bs (https://aica.guru/assets/main-DQ0H7cAg.js:2:59470)\n    at va (https://aica.guru/assets/main-DQ0H7cAg.js:2:1882)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Ea (https://aica.guru/assets/main-DQ0H7cAg.js:2:3834)\n    at Us\n    at oh (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:7803)\n    at P3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:17443)\n    at T3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:24235)"}

[2026-03-05T18:12:42.362Z] {}

[2026-03-05T18:12:42.363Z] %c❌ [ErrorBoundary] ErrorBoundary caught an error: color: #EF4444; font-weight: bold {} {"componentStack":"\n    at ta (https://aica.guru/assets/AthletePortalView-seTo6EhC.js:9:24786)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at za (https://aica.guru/assets/main-DQ0H7cAg.js:2:27210)\n    at Va (https://aica.guru/assets/main-DQ0H7cAg.js:2:19100)\n    at f (https://aica.guru/assets/main-DQ0H7cAg.js:2:57450)\n    at Gp (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:4073)\n    at N3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:8728)\n    at Suspense\n    at Ma (https://aica.guru/assets/main-DQ0H7cAg.js:2:18658)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at Bs (https://aica.guru/assets/main-DQ0H7cAg.js:2:59470)\n    at va (https://aica.guru/assets/main-DQ0H7cAg.js:2:1882)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Ea (https://aica.guru/assets/main-DQ0H7cAg.js:2:3834)\n    at Us\n    at oh (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:7803)\n    at P3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:17443)\n    at T3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:24235)"}

[2026-03-05T18:12:44.523Z] {}

[2026-03-05T18:12:44.524Z] %c❌ [ErrorBoundary] ErrorBoundary caught an error: color: #EF4444; font-weight: bold {} {"componentStack":"\n    at ta (https://aica.guru/assets/AthletePortalView-seTo6EhC.js:9:24786)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at za (https://aica.guru/assets/main-DQ0H7cAg.js:2:27210)\n    at Va (https://aica.guru/assets/main-DQ0H7cAg.js:2:19100)\n    at f (https://aica.guru/assets/main-DQ0H7cAg.js:2:57450)\n    at Gp (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:4073)\n    at N3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:8728)\n    at Suspense\n    at Ma (https://aica.guru/assets/main-DQ0H7cAg.js:2:18658)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Vm (https://aica.guru/assets/module-journey-BQI7qcGj.js:13:6133)\n    at Bs (https://aica.guru/assets/main-DQ0H7cAg.js:2:59470)\n    at va (https://aica.guru/assets/main-DQ0H7cAg.js:2:1882)\n    at Qm (https://aica.guru/assets/module-journey-BQI7qcGj.js:1:2738)\n    at Ea (https://aica.guru/assets/main-DQ0H7cAg.js:2:3834)\n    at Us\n    at oh (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:7803)\n    at P3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:17443)\n    at T3 (https://aica.guru/assets/vendor-react-Tn1wb__B.js:51:24235)"}
```

---

*Reported via Callout*

--- a/src/modules/flux/views/AthletePortalView.tsx
+++ b/src/modules/flux/views/AthletePortalView.tsx
@@ -176,6 +176,33 @@ export default function AthletePortalView() {
     finally { setUpdating(null); }
   }, [refetch]);
 
+  // ── Hooks that MUST be above early returns (React Rules of Hooks) ──
+
+  const pastWorkoutDays = useMemo(() => {
+    const micro = profile?.active_microcycle;
+    if (!micro?.slots?.length || !micro.start_date) return 0;
+    const today = new Date();
+    today.setHours(23, 59, 59, 999);
+    const startDate = new Date(micro.start_date);
+    let count = 0;
+    for (const slot of micro.slots) {
+      const weekOffset = (slot.week_number - 1) * 7;
+      const dayOffset = slot.day_of_week - 1;
+      const slotDate = new Date(startDate);
+      slotDate.setDate(startDate.getDate() + weekOffset + dayOffset);
+      if (slotDate <= today) count++;
+    }
+    return count;
+  }, [profile?.active_microcycle]);
+
+  const prescribedModalities = useMemo((): Array<keyof typeof MODALITY_CONFIG> => {
+    const mod = profile?.modality as keyof typeof MODALITY_CONFIG;
+    if (mod === 'triathlon') {
+      return ['triathlon', 'swimming', 'running', 'cycling'];
+    }
+    return mod && MODALITY_CONFIG[mod] ? [mod] : ['strength'];
+  }, [profile?.modality]);
+
   // ── Early returns ──
 
   if (isLoading) {
@@ -257,35 +284,7 @@ export default function AthletePortalView() {
 
   const micro = profile.active_microcycle;
   const modalityConfig = MODALITY_CONFIG[profile.modality];
-
-  // Count past workout days (days that have already passed) instead of manual completions
-  const pastWorkoutDays = useMemo(() => {
-    if (!micro?.slots?.length || !micro.start_date) return 0;
-    const today = new Date();
-    today.setHours(23, 59, 59, 999);
-    const startDate = new Date(micro.start_date);
-    let count = 0;
-    for (const slot of micro.slots) {
-      const weekOffset = (slot.week_number - 1) * 7;
-      const dayOffset = slot.day_of_week - 1;
-      const slotDate = new Date(startDate);
-      slotDate.setDate(startDate.getDate() + weekOffset + dayOffset);
-      if (slotDate <= today) count++;
-    }
-    return count;
-  }, [micro?.slots, micro?.start_date]);
   const completionPct = micro ? Math.round((pastWorkoutDays / Math.max(micro.total_slots, 1)) * 100) : 0;
-
-  // Derive athlete modalities from profile.modality
-  // Template categories (warmup, main, cooldown) don't map to training modalities,
-  // so we use the athlete's declared modality and expand triathlon to its components.
-  const prescribedModalities = useMemo((): Array<keyof typeof MODALITY_CONFIG> => {
-    const mod = profile.modality as keyof typeof MODALITY_CONFIG;
-    if (mod === 'triathlon') {
-      return ['triathlon', 'swimming', 'running', 'cycling'];
-    }
-    return MODALITY_CONFIG[mod] ? [mod] : ['strength'];
-  }, [profile.modality]);
 
   const weeks = micro
     ? [1, 2, 3, 4].map((wk) => {

